code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -20,80 +20,46 @@ adjustFireLoss(-(maxHealth / 70 + 0.5 + (maxHealth / 70) * recovery_aura/2)*(m))
 
 	..()
 
-	switch(stat)
-		if(DEAD) //Dead, nothing else to do.
-			return
-		if(UNCONSCIOUS)
-			if(is_zoomed)
+	if(stat == DEAD) //Dead, nothing else to do.
+		return
+	if(stat == UNCONSCIOUS)
+		if(is_zoomed)
+			zoom_out()
+	else
```

Comment:
Indentation.

Reply:
I don't seem to see any indentation problem here. But I may be wrong. If so, give me a hand.